README.md

```diff
@@ -22,6 +22,8 @@ docker build -t drone_manager_image .
 docker run -d --name dronemanager -p 8080:8080 drone_manager_image
 ```
 
+You can set the environment variable `DM_DRONE_VISION_RADIUS` to configure the vision radius of the drones.
```

Comment:
What units are these in? is it degrees?

Reply:
It's in whatever units your coordinates are in; which, now you mention it, isn't very useful if we're using degrees b/c that's gonna be a tiny number. Do we want to introduce unit conversion into the service? It wouldn't be that hard.

Reply:
Im happy with keeping it as degrees or switching to km and then converting, id lean towards keeping it as degrees just because it's less work 😅

Reply:
Cool, let's leave it at that